.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "wsrep-API/v26"]
 	path = wsrep-API/v26
-	url = https://github.com/percona/wsrep-API.git
-	branch = percona-4.x-8.0
+	url = https://github.com/kamil-holubicki/wsrep-API.git
+	branch = PXC-3789
```

Comment:
This should be avoided?

Reply:
Yes @satya-bodapati . This is to allow Jenkins to build the project. Once wsrep-API is approved and merged. I will change this to point to the official repo.

Reply:
ok, go ahead then